src/garage/sampler/vec_worker.py

```diff
@@ -0,0 +1,209 @@
+"""Worker that "vectorizes" environments."""
+import collections
+import copy
+
+import gym
+import numpy as np
+
+from garage import TrajectoryBatch
+from garage.sampler.env_update import EnvUpdate
+from garage.sampler.worker import DefaultWorker
+
+
+class VecWorker(DefaultWorker):
+    """Worker with a single policy and multiple environemnts.
+
+    Alternates between taking a single step in all environments and asking the
+    policy for an action for every environment. This allows computing a batch
+    of actions, which is generally much more efficient than computing a single
+    action when using neural networks.
+
+    Args:
+        seed(int): The seed to use to intialize random number generators.
+        max_path_length(int or float): The maximum length paths which will
+            be sampled. Can be (floating point) infinity.
+        worker_number(int): The number of the worker this update is
```

Comment:
I feel like this is better named `worker_index`

Reply:
Yeah, maybe. But it's out of scope for this PR either way, since that's part of the worker API.